go/vt/vtgate/engine/ordered_aggregate.go

```diff
@@ -58,6 +58,10 @@ type OrderedAggregate struct {
 	// from the result received. If 0, no truncation happens.
 	TruncateColumnCount int `json:",omitempty"`
 
+	// Collations stores the collation ID per column offset.
+	// It is used for grouping keys and distinct aggregate functions
+	Collations map[int]collations.ID
+
```

Comment:
should we not remove from GroupByParams?
This can be used instead of keeping in GroupByParams or AggregateParams

Reply:
it's kind of nice to keep it in those data structures, since it makes the description printing much easier. do you feel strongly about this?

Reply:
it is just duplicated information. future changes might miss updating in one of them.

Reply:
well, it is duplicated, but since the map is built from the `GroupByParams` and `AggregateParams` objects, and they are helpful as they are when printing the plan description, I thought it was nice to keep the collation info there